packages/cli/src/commands/env/index.ts

```diff
@@ -111,6 +112,7 @@ export default async function main(client: Client) {
     argv = getArgs(client.argv.slice(2), {
       '--yes': Boolean,
       '-y': '--yes',
+      '--environment': String,
```

Comment:
```suggestion
      '--environment': String,
      '--env': '--environment',
```

I think adding a `--env` alias for this is pretty reasonable. We could add that alias to `vc pull` as well.